src/shared/components/palette/palette.module.ts

```diff
@@ -174,7 +176,6 @@ import { WriteCaseFlagFieldComponent } from './case-flag';
     WriteEmailFieldComponent,
     WriteDateFieldComponent,
     WriteDateContainerFieldComponent,
-    WriteCaseFlagFieldComponent
```

Comment:
@johnbenjamin-hmcts - `WriteCaseFlagFieldComponent` is no longer being exported... is that intentional? (Maybe it doesn't need to be exported.)

Reply:
The `WriteCaseFlagFieldComponent` and `ReadCaseFlagFieldComponent` are bundled inside `CaseFlagModule` and this module is imported in the `PaletteModule`.